prow/cluster/cherrypicker_deployment.yaml

```diff
@@ -24,6 +24,7 @@ spec:
         - --dry-run=false
         - --use-prow-assignments=false
         - "--labels=cla: yes"
+        - --labels=release-notes-none
```

Comment:
Is the --labels going to cause an issue?

Reply:
It looks like GitHub-endpoint has multiple.

Reply:
on the contrary. It only works this way. Comma separated will be interpreted as only one string.

Reply:
Played locally with https://github.com/kubernetes/test-infra/blob/master/prow/external-plugins/cherrypicker/main.go#L65 and https://github.com/kubernetes/test-infra/blob/master/prow/flagutil/strings.go

Reply:
Result of `fmt.Printf("%d - %v\n", len(o.labels.Strings()), o.labels.Strings())`:
```
✔ ~/src/k8s/test-infra/prow/external-plugins/cherrypicker [master|✚ 1] 
12:06 $ go run main.go --labels=x1,x2,x3
1 - [x1,x2,x3]

✔ ~/src/k8s/test-infra/prow/external-plugins/cherrypicker [master|✚ 1] 
12:21 $ go run main.go --labels=x1 --labels=x2 --labels=x3
3 - [x1 x2 x3]
```

Reply:
Thanks

Reply:
> It looks like GitHub-endpoint has multiple.

yeah, they use the same backend to parse array of strings

Reply:
Thanks.